modules/ballerina-core/src/main/resources/MessagesBundle.properties

```diff
@@ -21,7 +21,9 @@ invalid.type = invalid type ''{0}''
 incompatible.types = incompatible types: expected ''{0}'', found ''{1}''
 redeclared.symbol = redeclared symbol ''{0}''
 incompatible.types.cannot.convert = incompatible types: ''{0}'' cannot be converted to ''{1}''
+cannot.convert.with.suggestion = incompatible types: ''{0}'' cannot be converted to ''{1}'', try casting
```

Comment:
wouldn't be good to suggest the notation as well?
eg: `try using casting with <string>... `